authentication-guides/workday/README.md

```diff
@@ -0,0 +1,240 @@
+
+![https://developer.moveworks.com/static/2a3fbc8565c2adb8678d2388338278d6/c6bbc/gen_refresh_token.png](https://developer.moveworks.com/static/2a3fbc8565c2adb8678d2388338278d6/c6bbc/gen_refresh_token.png)
+
+Note your  new refresh token.
+
+![https://developer.moveworks.com/static/48972e313711f799a7f8172586f256ca/c6bbc/refresh_token.png](https://developer.moveworks.com/static/48972e313711f799a7f8172586f256ca/c6bbc/refresh_token.png)
+
+
+
+## Test the Connection
+
+To test the connection, we need to first use the client_id and client_secret against the token api to get a bearer token. We then use that bearer token to run a query against the api.
+
+We will use curl to do this, if you are using a Linux or Apple computer, curl is included and can be accessed from the terminal. With Windows, you will need to [download](https://curl.se/windows/) it first.
+
+### Get a Bearer Token
+
+In the below snippet, replace YOUR_CLIENT_ID, YOUR_CLIENT_SECRET, YOUR_REFRESH_TOKEN and YOUR_TENANT_NAME with the values from the above steps
+```
+
+curl -X POST -H "Content-Type: application/x-www-form-urlencoded" -d "client_id=YOUR_CLIENT_ID&client_secret=YOUR_CLIENT_SECRET&grant_type=refresh_token&refresh_token=YOUR_REFRESH_TOKEN" https://wd2-impl-services1.workday.com/ccx/oauth2/YOUR_TENANT_NAME/token
+```
+
+You should get a response back that looks similar to 
+```json
+{ "access_token": "eyJ0eXAiOiJhdCtKV1QiLCJhbGciOiJSUzUxMiJ9.eyJzdWIiOiIzZWJlNjkyNzVmMmE0MTk3OWY0M2EwNGQ1YTYxY2FiMyIsImF1ZCI6Imh0dHBzOi8vd2QyLWltcGwtc2VydmljZXMxLndvcmtkYXkuY29tL2NjeC8iLCJzY29wZSI6Im9hdXRoIiwiaXNzIjoiaHR0cHM6Ly93ZDItaW1wbC1zZXJ2aWNlczEud29ya2RheS5jb20vY2N4L2FwaS92MS9iYWluY2FwaXRhbF9nbXM0IiwiZXhwIjoxNzA0NDc0NDM4LCJpYXQiOjE3MDQ0NjAwMzgsImp0aSI6ImFuRjNaMk51TURkdlpqWnFiM055Y0ROMllXUjFhRFZ6ZEhsNloybDBjekpoWkdGbFpEVm1PVzV2ZURsb2FESjRkRE0zZDJWMFpXcG5NV0ZyZUhWaGVEbG1hM1EzT0RSa1lucGhhM2x3ZG5ONVpqbHRkM1k1YW1VNU1uZHVPV00xT1c5MExqZzBNbUV5WlRrekxUa3hZekV0TkRnNFlpMWhaRGRrTFdObU5HVmtZMkUxWVdRM1pRPT0iLCJjbGllbnRfaWQiOiJZekkzWTJVME9ETXRPVEE0TVMwMFlXTXhMVGt4T0RndE0yRmxPRGcwTjJObU5tRmgifQ.UN7cYzDio6lL4-2XuVjpofp54gB9YmPrLi4ZQF7F98DBrgcUoo3RFzp7Z2mV5a3dCVUMItsiRARN0y35CfLpvrOLrm4XHKi2h69Q1vud2UayZAbk2GWQ5b-wE3IPRvYsPlZNDhH0zND8t8EZL4U5yS_Mf4-84GrAqZrVP0_SyLYAEHkkz33LHc5df96JLn_Xva8CdgYvlGU1NKTIYItGjyXvutKfLV_9qybA-sJ8BZGkK2krDqhg20HUVD8unHV4ZO9gfzN7BtRvRZJb7-Gol-eK346sg309831xvwLSE5FiGsWYMk1srZx0fJQBI85HzVV-PBmIz5MjraaQkb30ww", "token_type": "Bearer", "refresh_token": "ee8enxymxyuom832m8rb5aoh3790lw4f44hb6wabuojlzwdlfxq52mw0tutxtvl7b6c8dhztncj5ymeeggppr6q3wiznvgxvean" }
+```
+
+You will want to pull out the value for access_token to use it for the quick test.
+
+### Test a Query
+ This is a simple WQL query to get five employees (in no particular order) from Workday.
+
+Replace YOUR_TENANT_NAME and YOUR_BEARER_TOKEN in the below with the values from previous steps.
+:
+
+```
+curl --location 'https://wd2-impl-services1.workday.com/ccx/api/wql/v1/YOUR_TENANT_NAME/data?limit=5&offset=0' \
```

Comment:
Any reason why curl instead of Postman. Postman is a little more accessible to folks and matches what we do in the other guides.

Not required, just curious